pytorch_lightning/accelerators/ddp_cpu_spawn_accelerator.py

```diff
@@ -74,10 +74,12 @@ def train(self):
     def ddp_train(self, process_idx, mp_queue, model):
         """
         Entry point for ddp
+
         Args:
             process_idx:
             mp_queue: multiprocessing queue
             model:
+
         Returns:
         """
```

Comment:
mind add something here?

Reply:
Function returns nothing, so it should probably just be removed

Reply:
@rohitgr7 Thanks for your response! @SkafteNicki Agreed.

Reply:
I meant `process_idx` and `model` :]

Reply:
@rohitgr7 Thank you for the review and clarification! I am not familiar with distributed training, and thus I don't think I can update the docstring properly. Can you help if you know what to add to those variables?